src/webgpu/api/validation/image_copy/image_copy.ts

```diff
@@ -4,11 +4,28 @@ import {
   SizedTextureFormat,
   kTextureFormatInfo,
   isCompressedTextureFormat,
+  kCompressedTextureFormats,
+  kDepthStencilFormats,
 } from '../../../format_info.js';
 import { align } from '../../../util/math.js';
 import { ImageCopyType } from '../../../util/texture/layout.js';
 import { ValidationTest } from '../validation_test.js';
 
+export const kReducedFormatsList = [
+  'r8unorm',
```

Comment:
For testing compat copyT2B workaround impl purporse I'd like to also include ['r8snorm', 'rg8snorm', 'bgra8unorm']